doc/classes/DisplayServer.xml

```diff
@@ -1694,7 +1697,8 @@
 			<return type="int" />
 			<param index="0" name="screen" type="int" default="-1" />
 			<description>
-				Returns the dots per inch density of the specified screen. If [param screen] is [constant SCREEN_OF_MAIN_WINDOW] (the default value), a screen with the main window will be used.
+				Returns the dots per inch density of the specified screen. Returns [code]72[/code] if [param screen] is invalid.
```

Comment:
The default on Android will be different if the parameter is invalid. Maybe we should remove that it returns `72` here, and just leave the last note which specifies that it returns `72` on unsupported platforms.